package.json

```diff
@@ -24,6 +24,7 @@
     "react-avatar": "^2.5.1",
     "react-bootstrap": "^0.32.1",
     "react-dom": "^16.3.2",
+    "react-helmet": "^5.2.0",
```

Comment:
@muhozi What is this library doing?

Reply:
@alexmochu It helps to manage header contents including `title`, `description` and `meta tags` of the page.